Scripts/Src/Program.cs

```diff
@@ -4,94 +4,96 @@
                     {
-                        contents = contents.Replace($@"### [Kickoff] Application Setup{endOfLine}{endOfLine}","")
-                                            .Replace($@"After cloning the bootstrap, follow the [kickoff guide](https://github.com/Wolox/tech-guides/blob/master/net-core/docs/kickoff/README.md#kickoff).{endOfLine}","")
-                                            .Replace($@"And happy coding!{endOfLine}","");
+                        contents = contents.Replace($@"### [Kickoff] Application Setup{endOfLine}{endOfLine}", string.Empty)
+                                            .Replace($@"After cloning the bootstrap, follow the [kickoff guide](https://github.com/Wolox/tech-guides/blob/master/net-core/docs/kickoff/README.md#kickoff).{endOfLine}", string.Empty)
+                                            .Replace($@"And happy coding!{endOfLine}", string.Empty);
                     }
-                    if(deleteAuth == DeleteAuthenticationParamValue())
+                    if (deleteAuth == DeleteAuthenticationParamValue())
                     {
-                        if(file.Contains("/Startup.cs"))
+                        if (file.Contains("/Startup.cs"))
                         {
                             Console.WriteLine("Updating Startup.cs ...");
+
                             // We use spaces to delete indentation
-                            contents = contents.Replace($@"using Microsoft.AspNetCore.Identity.EntityFrameworkCore;{endOfLine}", "")
-                                                .Replace($@"using Microsoft.AspNetCore.Identity;{endOfLine}", "")
-                                                .Replace($@"using Microsoft.AspNetCore.Authentication.Google;{endOfLine}", "")
-                                                .Replace($@"app.UseAuthentication();{endOfLine}            ", "");
+                            contents = contents.Replace($@"using Microsoft.AspNetCore.Identity.EntityFrameworkCore;{endOfLine}", string.Empty)
+                                                .Replace($@"using Microsoft.AspNetCore.Identity;{endOfLine}", string.Empty)
+                                                .Replace($@"using Microsoft.AspNetCore.Authentication.Google;{endOfLine}", string.Empty)
+                                                .Replace($@"app.UseAuthentication();{endOfLine}            ", string.Empty);
                             startIndex = contents.IndexOf("//Begin for Identity");
                             endIndex = contents.IndexOf("//Final for Identity", startIndex + 1);
-                            contents = contents.Remove(startIndex, (endIndex - startIndex));
-                            contents = contents.Replace($"//Final for Identity;{endOfLine}", "");
+                            contents = contents.Remove(startIndex, endIndex - startIndex);
+                            contents = contents.Replace($"//Final for Identity;{endOfLine}", string.Empty);

```

Comment:
`contents = contents.Replace($"// Final for Identity;{endOfLine}", string.Empty);`